### ISSUE TITLE
FR: Nav & Menu add options variable to sub items

### ISSUE THREAD

**Issue by kullar84** [yiisoft/yii2#5629]
Please add options to subitems so we could add class to sub ul tag.

``` php5
if ($items !== null) {
            $linkOptions['data-toggle'] = 'dropdown';
            Html::addCssClass($options, 'dropdown');
            Html::addCssClass($linkOptions, 'dropdown-toggle');
            $label .= ' ' . Html::tag('b', '', ['class' => 'caret']);
            if (is_array($items)) {
                if ($this->activateItems) {
                    $items = $this->isChildActive($items, $active);
                }
                $items = Dropdown::widget([
                    'items' => $items,
                    'encodeLabels' => $this->encodeLabels,
                    'clientOptions' => false,
                    'options' => $options,
                    'view' => $this->getView(),
                ]);
            }
 }

```

**Comment by spikyjt** [comment]
I think this should be available for as many levels as it can render. Just because navbar/dropdown don't do second level pop-outs, doesn't mean you can't display extra levels of menu indented or however you like. This should apply to `\yii\widgets\Menu` (as per Yii 1.1 CMenu) and `\yii\bootstrap\Dropdown` (#7480) as well.

**Comment by cebe** [comment]
keeping this open for `yii\widgets\Menu`

**Comment by samdark** [comment]
Won't be implemented in 2.0.
